core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -708,5 +707,195 @@ namespace ledger {
             return _explorer->getFees().callback(getContext(), callback);;
         }
 
+        FuturePtr<api::BitcoinLikeTransaction> BitcoinLikeAccount::getReplayByFeeTransaction(const std::string & hash, const std::shared_ptr<api::Amount> & additionaFeesPerBytes, bool forceBuild) {
```

Comment:
As I understand forceBuild is here only for testing purpose.
I am not a big fun of approach to include testing logic in "real" code, because it makes the "real" code harder to reason about, and open the door for new errors.
What we can do here is write a function that create RBF transaction having other transaction as input. So in tests we can get some transaction and directly call this new function and in "real" code first get transaction, then check the transaction and then call the `createRBFFromTransaction(checkedTransaction)`

Reply:
Hmm if we do that we'll have to go through inputs twice, since we still need the inputs' value, unless I didn't really get what you want to say ? 🤔 
And here we are talking about a boolean which explicitly means what it does and used at only one condition so not adding that much complexity, if there were too much branching and additional parameters I would agree but it still to be discusses VS adding other methods with different signatures 🤔